src/lib_ccx/ccx_encoders_spupng.c

```diff
@@ -21,7 +21,13 @@
 #define CANVAS_WIDTH 600
 
 FT_Library  ft_library = NULL;
+
+// The FT_Face object handles typographical information
+// The different face variables are for the regular, bold, italic, etc. fonts
 FT_Face	 face = NULL;
+FT_Face	 faceb = NULL;
+FT_Face	 facei = NULL;
+FT_Face	 facez = NULL;
```

Comment:
The variable names don't make much sense. Consider renaming them to something like `face_bold_italics`.

Reply:
Many truetype fonts use the z suffix to denote a .ttf file for the bold and italic version of a font, especially in Windows (e.g regular = calibri.ttf, italics =calibrii.ttf, bold = calibrib.ttf, bold & italic = calibriz.ttf). I just thought the z suffix was something that people used to denote italics and bold. I'll change the variable names if they're too confusing.